src/basics/strings/word-distance/index.js

```diff
@@ -7,6 +7,21 @@
  * distance('abc', 'adc') //=> 1
  * distance('abc', 'cde') //=> 3
  */
-function distance(word) {}
+function distance(wordA,wordB) {
+    var counter = 0 ;
+    var len = wordA.length ;
+    for (let i = 0; i < len ; i++) {
+        for (let j = i; j < len; j++) {
+            if (wordA[i]==wordB[j]) {
+                break ;
+            }else{
+                counter += 1 ;
+                break;
+            }
+            
+        }
+        
+    }return counter 
+}
```

Comment:
if we define `N` to be the length of wordA, what is the time complexity of this function?
Can we make it `O(N)`?

Reply:
the complexity would be O(n^2)
I think we can by using only one loop